### DISCUSSION TITLE
Duplicate meeting lists

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#646] [Q&A] [answered]
I seem to have created a duplicate set of all our meetings on our Area website (currently in staging).
Originally I used a CSV file of the meetings from our live website during development.
We are nearing go-live so I recently copied the URL (generated by clicking on the data source links) one by one and then added those current data sources.

But now I have two copies of _nearly_ every meeting:
- 3,349 Live site
- 6,519 Staging site

Seems I've got one from the CSV import (a somewhat older list), and the current feed, but they've combined ..?
I've tried to "Bulk Action > Delete" but that only moves a screen-full of meetings to the Trash. :(
_Is there a way to delete just the CSV data?_

I've attached images of our live site, staging site, and duplicate listing for visual reference.
Any help would be greatly appreciated.

**Thank you!**
scott_fritzinger@me.com
webservant@area59aa.org

![Current-Live-Site-01-17-22](https://user-images.githubusercontent.com/95769327/149830188-af235250-30b4-4673-a1f2-79dc7a3d6dd5.png)
![Current-Staging-Site-01-17-22](https://user-images.githubusercontent.com/95769327/149830193-0555d525-4efa-4fa8-a439-f585c03351ca.png)
![Staging-Site-DUPLICATE-Meetings-01-17-22](https://user-images.githubusercontent.com/95769327/149830194-d8d122b6-cb62-4eda-aa97-b60abb0ec584.png)

**Comment by lovethecodelabs** [comment]
If no other wordpress modifications were made,  a sql query on the wp-posts table with modifications after a date time to be deleted would work

**Comment by brianw-area24** [comment, marked as answer]
I'm hoping all your meetings come from the feeds. If so, there is a pretty easy answer. If not, then don't do what I'm suggesting here because this will delete those additional meetings...

If all your meetings come from feeds, try this.

- Download your meetings in CSV format
- Modify this CSV file, and delete all but 1 meeting
- Import this modified CSV file, deleting all meetings, locations, and districts
- Manually delete this one meeting
- Refresh all your feeds

This is an easy way to remove all your meetings, then refresh them from the feeds. However, if your have meetings don't come from the feeds, this will delete those meetings also.

Brian W

**Comment by fritzisa** [comment]
Thank you Brian,

That was a very effective solution.

A appreciate your help!

Scott Fritzinger  |  Area 59 Web Servant  |  (610) 762-7792
